### PR TITLE
Modified space bubble entry effect

### PR DESCRIPTION
**Pull request by zfox23**
Using this PR, avatars that enter your bubble will freeze for a moment before fading. This is to potentially give users a better idea of what's happening when they enter your bubble.

### CODE REVIEW

File: interface/src/avatar/AvatarManager.cpp [hunk 1]
```diff
@@ -255,13 +255,22 @@ void AvatarManager::handleRemovedAvatar(const AvatarSharedPointer& removedAvatar
...
+    if (removalReason == KillAvatarReason::TheirAvatarEnteredYourBubble) {
+        // Show the bubble on the client whose bubble was entered
+        emit DependencyManager::get<UsersScriptingInterface>()->enteredIgnoreRadius();
+        // Freeze the avatar that entered the bubble for a moment, then make them fade
+        QTimer* freezeTimer = new QTimer(this);
+        QObject::connect(freezeTimer, &QTimer::timeout, [=] {
+            _avatarFades.push_back(removedAvatar);
+            delete freezeTimer;
+        });
+        freezeTimer->start(3000);
```
Comment (ZappoMan) [inline interface/src/avatar/AvatarManager.cpp]: coding standard: no magic numbers

Comment (zfox23) [inline interface/src/avatar/AvatarManager.cpp]: I'll definitely be fixing this (and improving the behavior) if we determine we want to go in this direction for the bubble effect - but I'll just go ahead and fix this now. Thanks.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 53e6bd9ba03650ab065ef6bdafd69c7d92f533e2 available at: https://gist.github.com/7a2d6ea482e86e4ddce19c4723209837<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9338-53e6bd9.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9338-53e6bd9.dmg

**Comment by zfox23** [comment]
I changed the code so that there are no longer duplicate avatars, but now the implementation is pretty dirty - it creates and deletes hundreds of QTimers upon avatar fade. If we decide this is the way to go with the bubble effect, I'll clean up the implementation.

**Comment by hifi-gustavo** [comment]
Build console output for commit: e8af4e62222381ef9749ed1c972727cb9e49a412 available at: https://gist.github.com/7a2d6ea482e86e4ddce19c4723209837<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9338-e8af4e6.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9338-e8af4e6.dmg

**Comment by howard-stearns** [comment]
I guess we can re-open as needed.

**Pull request closed without merging**